node/node.go

```diff
@@ -90,7 +74,7 @@ func NewNodeController(p NodeProvider, node *corev1.Node, nodes v1.NodeInterface
 		n.statusInterval = DefaultStatusUpdateInterval
 	}
 
-	n.nodePingController = newNodePingController(n.p, n.pingInterval, n.pingTimeout)
+	n.nodePingController = nodepingcontroller.NewNodePingController(n.p, n.pingInterval, n.pingTimeout)
```

Comment:
A linter must be catching this as stutter. If this is really the package name we're going with, at least `nodepingcontroller.New()` would make more sense to me.